content/kots-cli/restore-ls.md

```diff
@@ -0,0 +1,25 @@
+---
+date: 2019-10-30
+linktitle: "restore ls"
+title: kots restore ls
+weight: 90230
+---
+
+The `kots restore ls` shows a list of all the available instance restores for disaster recovery. 
```

Comment:
Hmm... not sure what to call these. "list all running and completed restores" or "restorations"? I guess we can go with this as is.